### PR TITLE
Social theme cookies and privacy page

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
Added cookies and privacy page for the Social theme.
Also made some slight modification to existing cookies and privacy (default theme) content for consistency.

### How to review 
Ensure layout and content matches whats on [Trello](https://trello.com/c/0HZdt4MN/2448-cookies-and-privacy-for-social-theme)

### Checklist

* [x] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Comment by dcdarrell9** [comment]
screen shot 2018-10-04 at 13 40 31
Doesn't look like it matches the card

**Comment by dcdarrell9** [comment]
screen shot 2018-10-04 at 13 42 13
Same here, should there be a 'purpose' and 'description'?

File: app/translations/messages.pot [hunk 1]
```diff
@@ -482,18 +466,6 @@ msgstr ""
 msgid "Save and sign out"
 msgstr ""
 
-#: app/templates/partials/answers/date.html:5
```
Comment (jonnyshaw89) [inline app/translations/messages.pot]: Why are these removed?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Comment by MebinAbraham** [comment]
> screen shot 2018-10-04 at 13 40 31
> Doesn't look like it matches the card

Updated.

**Comment by MebinAbraham** [comment]
> screen shot 2018-10-04 at 13 42 13
> Same here, should there be a 'purpose' and 'description'?

@dcdarrell9 The header of the table was being hidden purposely?, not sure, I removed the class that hid it from both default and social theme.

File: app/translations/messages.pot (hunk 1 above)
Comment (MebinAbraham) [reply app/translations/messages.pot]: Babel was being a right **#@&%^!**. Fixed now.

File: app/templates/static/cookies-privacy.html [hunk 2]
```diff
@@ -373,28 +375,40 @@ <h2 class="question__title neptune">
...
                                 </h2>
                                 <div class="question__description mars">
-                                    <p>{{ _("We welcome the opportunity to be able to answer any questions or concerns you may have around the information provided here. If you are unhappy with our response to any of your requests or you have concerns around how your data has been handled, you have the right to lodge a complaint with the Information Commissioner’s Office") }}.</p>
-                                    <p>{{ _("Information Commissioner’s Office") }}<br>Wycliffe House<br>Water Lane<br>Wilmslow<br>Cheshire<br>SK9 5AF</p>
+                                    <p>{{ _("We welcome the opportunity to be able to answer any questions or concerns you may have around the information provided here. If you are unhappy with our response to any of your requests or you have concerns around how your data has been handled, you have the right to lodge a complaint with the Information Commissioner’s Office.") }}</p>
+                                        {{ _("Information Commissioner’s Office") }}<br>
+                                        {{ _("Wycliffe House") }}<br>
```
Comment (jonnyshaw89) [inline app/templates/static/cookies-privacy.html]: Do we want to translate addresses? this is an english address?

File: app/themes/social/templates/static/cookies-privacy.html [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+                                <br>
+
+                                <h2 class="question__title neptune">
+                                    {{ _("Data controller") }}
+                                </h2>
+                                <div class="question__description mars">
+                                    <p>{{ _("The Data Controller for ONS Surveys is") }}:</p>
+                                    <p>
+                                        {{ _("Office for National Statistics") }}<br>
+                                        {{ _("Government Buildings") }}<br>
```
Comment (jonnyshaw89) [inline app/themes/social/templates/static/cookies-privacy.html]: Same here re translating addresses

File: app/themes/social/templates/static/cookies-privacy.html [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+                                </div>
+
+                                <h2 class="question__title neptune">
+                                    {{ _("Contacting the Information Commissioner’s Office") }}
+                                </h2>
+                                <div class="question__description mars">
+                                    <p>{{ _("We welcome the opportunity to be able to answer any questions or concerns you may have around the information provided here. If you are unhappy with our response to any of your requests or you have concerns around how your data has been handled, you have the right to lodge a complaint with the Information Commissioner’s Office.") }}</p>
+                                    <p>
+                                        {{ _("Information Commissioner’s Office") }}<br>
+                                        {{ _("Wycliffe House") }}<br>
```
Comment (jonnyshaw89) [inline app/themes/social/templates/static/cookies-privacy.html]: another address

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Review by insacuri — APPROVED** [review]
👍 LGTM (after the address translation issue gets sorted out)

**Comment by MebinAbraham** [comment]
@jonnyshaw89 I'm not sure whether we should be marking addresses for translation. We have some addresses marked for translation and some not, so I was just trying to make 1 consistent. And the contact us page for the social theme you did recently have address marked for translation so thought maybe there was a reason you kept it. I'll ask Simon what we should do.

**Comment by jonnyshaw89** [comment]
@MebinAbraham @charlotte-hirst has clarified that Addresses should not be translated

**Comment by MebinAbraham** [comment]
@jonnyshaw89 Ready for re-review.

**Review by jonnyshaw89 — APPROVED** [review]

**Pull request merged**